test/source/tests/setup.ts

```diff
@@ -436,6 +446,60 @@ AN8G3r5Htj8olot+jm9mIa5XLXWzMNUZgg==
       await gmailPage.notPresent(['@webmail-notification']);
     }));
 
+    ava.default('mail.google.com - success notif after setup, click hides it, does not re-appear + offers to reauth', testWithBrowser('compatibility', async (t, browser) => {
+      const acct = 'flowcrypt.compatibility@gmail.com';
+      const gmailPage = await browser.newPage(t, TestUrls.mockGmailUrl(), undefined, await getAuthorizationHeader(t, browser, acct));
```

Comment:
can substitute with the shortcut `const gmailPage = await openMockGmailPage(t, browser, acct);`

Reply:
or something like 
`await browser.newPage(t, TestUrls.mockGmailUrl(), undefined, { Authorization: 'Bearer no-access-for-extension'});` or some other dummy descriptive token might work